src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ObservationMapperTest.java

```diff
@@ -193,6 +194,30 @@ void testMapLaboratoryInvestigationCodedResult() {
             ));
     }
 
+    @Test
+    void testMapInterpretation() {
+        final var message = new Message(List.of(
+            "S02+02", // ServiceReportDetails
+            "S06+06", // InvestigationSubject
+            "GIS+N", // LabResult
+            "INV+MQ+c:911::d", // LaboratoryInvestigation
+            "RFF+ASL:1", // Reference
+
+            "RSL+NV+1.23:7++:::units+HI" // LaboratoryInvestigationResult
+        ));
+
+        final var observations = mapper.mapToObservations(message, mockPatient, emptyMap(), mockOrganization,
+            mockPractitioner);
+
+        assertThat(observations).hasSize(1)
+            .first()
+            .extracting(Observation::getInterpretation)
+            .extracting(CodeableConcept::getText)
+            .satisfies(result ->
+                assertThat(result).isEqualTo(DeviatingResultIndicator.ABOVE_HIGH_REFERENCE_LIMIT.getCode()
+                    + " : " + DeviatingResultIndicator.ABOVE_HIGH_REFERENCE_LIMIT.getDescription()));
```

Comment:
Can use `.isEqualTo` directly on an AbstractObjectAssert - no need to go through `.satisfies`. Also, I'd prefer just to see the raw string being put in the FHIR output.

```suggestion
        assertThat(observations).hasSize(1)
            .first()
            .extracting(Observation::getInterpretation)
            .extracting(CodeableConcept::getText)
            .isEqualTo("HI : Above high reference limit");
```